docs/exercise008-bonus-activity.md

```diff
@@ -0,0 +1,83 @@
+# Exercises 008 - Optional Bonus Activity!
+
+Congratulations for finishing these JavaScript challenges!
+
+You have covered so many different JavaScript concepts and skills in a very short space of time, including:
+- Problem Solving
+- For Loops
+- ES6 methods
+- Conditionals
+- Data Types
+- Objects
+and more!
```

Comment:
I like the re-enforcement here covering what they have learnt to say well done to them as well 👍